src/ui-kit/form/FieldText/FieldText.tsx

```diff
@@ -0,0 +1,53 @@
+import React from 'react'
+import { Field, FieldProps } from 'formik';
+
+import TextInput from 'components/layout/inputs/TextInput'
+
+
+interface IProps {
+  disabled?: boolean,
+  inputRef?: HTMLInputElement,
+  label: string,
+  maxLength?: number | null,
+  name: string,
+  placeholder?: string,
+  required?: boolean,
+  subLabel?: string,
+  type?: 'text' | 'email',
+}
```

Comment:
je pense qu'on n'est pas obligé de définir tous les attributs natifs des champs, à voir, mais un truc du genre : 



```suggestion
interface IFieldTextProps extends React.HTMLProps<HTMLInputElement> {
  inputRef?: HTMLInputElement,
  label: string,
  name: string,
  subLabel?: string,
  type?: 'text' | 'email',
}

const FieldText = ({ 
   inputRef,
   label, 
   name, 
   subLabel = '', 
   type = 'text',
   ...rest,
 }: IProps): JSX.Element => {...}
```